api/opds.py

```diff
@@ -245,20 +267,28 @@ def annotate_work_entry(self, work, active_license_pool, edition, identifier, fe
         for tag in link_tags:
             entry.append(tag)
 
-        # Add a link for related recommendations if available.
-        if NoveListAPI.is_configured():
+        # Add a link for related books if available.
+        if self.related_books_available(active_license_pool):
             feed.add_link_to_entry(
                 entry,
                 rel='related',
                 type=OPDSFeed.ACQUISITION_FEED_TYPE,
                 title='Recommended Works',
                 href=self.url_for(
-                    'recommendations',
+                    'related_books',
```

Comment:
This isn't the link relation, it's the method from `api/routes.py` for which the URL is being created. The "related" rel is set 4 lines up.